docs/source/usage/plugins/adios.rst

```diff
@@ -60,6 +60,45 @@ PIConGPU command line option description
    #. dump all species data each 128th time step, **do not create** the adios journal meta file.
    #. dump all field data each 1000th time step but **create** the adios journal meta file.
 
+.. _usage-plugins-ADIOS-meta:
+
+Meta Files
+^^^^^^^^^^
+
+Disabling meta (journal) file creation on-the-fly can improve output performance for large scale runs.
+Make sure to run ``bpmeta <theoretical-meta-fileName>`` after your simulation on created files in order to create the meta file in post-processing.
```

Comment:
optional:
confusing wording - run ... on should be closer together 
... on created files after your simulation ...